files/help/ares-device.help

```diff
@@ -57,17 +66,17 @@
             "# Display system information",
             "ares-device -i -d DEVICE",
             "",
-            "# Display session information",
-            "ares-device -s -d DEVICE",
-            "",
-            "# Display CPUs and memories usage with time interval",
+            "# Display resource usage with time interval",
```

Comment:
I think it need to explain what about "resource".
um.. "CPU and memory"  OR "system resource" ?

Reply:
Fixed